transcom/mymove
# B-18192 move query logic from handler to service object

Summary

Move query logic from handler to service object

How to test

1. Login as a customer
2. Create a PPM, submit it.
3. Login as a SC, fill in the required data, submit.
4.  Back as a customer, upload documents, add several Expenses.
5. Delete an Expense, and confirm the delete is successful.
